src/ctapipe/image/cleaning.py

```diff
@@ -112,6 +113,41 @@ def tailcuts_clean(
         )
 
 
+def bright_cleaning(image, threshold, fraction):
+    """
+    Clean an image by removing pixels below a fraction of the mean charge
+    in the 3 brightest pixels.
+
+    Select no pixels instead if the mean charge of the brightest pixels
```

Comment:
This is ambiguous. What does "select no pixels" mean? Is the mask all false? Or are no pixels removed?